src/renderer/components/+cluster/cluster-issues.tsx

```diff
@@ -37,21 +38,22 @@ export class ClusterIssues extends React.Component<Props> {
   private sortCallbacks = {
     [sortBy.type]: (warning: IWarning) => warning.kind,
     [sortBy.object]: (warning: IWarning) => warning.getName(),
-    [sortBy.age]: (warning: IWarning) => warning.age || "",
+    [sortBy.age]: (warning: IWarning) => warning.getTimeDiffFromNow(),
```

Comment:
Doesn't feel quite right, `getTimeDiffFromNow()` calls `Date.now()`, which means that for sorting a different "now" is used each time this is called. Could cause problems? Just use a static `timeDiffFromNow` in `IWarning`? Or we really just need something that converts the `age` string to a number (and done when `warnings` is populated)?

Reply:
So `age` is `getAge()` which calls `getTimeDiffFromNow()` anyway.

Reply:
Right, but it was not getting called during sorting. And also that means we shouldn't need to use `getTimeDiffFromNow()` directly.

Reply:
Fair good point.

Reply:
Though we also don't want it use a different "now" when creating it. So probably the best solution would be to have a `warning.getTimeDiffFrom(firstLoad)` where `this.firstLoad = Date.now()` is called in the `componentDidMount` method.

Reply:
Agreed as out of scope, though it shouldn't change the sorting order (too much).